content/en/agent/versions/_index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Agent Version differences
+kind: documentation
+further_reading:
+- link: "agent/versions/upgrade_to_agent_v7"
+  tag: "Documentation"
+  text: "Upgrade to Agent v7"
+- link: "agent/versions/upgrade_to_agent_v6"
+  tag: "Documentation"
+  text: "Upgrade to Agent v6"
+- link: "agent/faq/agent_v6_changes"
+  tag: "FAQ"
+  text: "Agent v6 Changes"
+---
+
+
+## Changes between major Agent versions
+
+{{< tabs >}}
+{{% tab "Agent v7 vs v6" %}}
+
+Agent v7 is the latest major version of the Datadog Agent. The only change from Agent v6 is that **this version only includes support for Python 3 for Integrations and Custom Checks**.
+
+See the [Upgrade to Agent v7 documentation][1] to learn how to upgrade your Agent to version 7. All official Integrations support Python 3 out-of-the-box. Follow the [Python 3 Custom Check Migration guide][2] to migrate your custom checks to Python 3.
+
+**Note**: You can test this migration with Agent v6, by [Using Python 3 with Datadog Agent v6][3].
+
+[1]: /agent/versions/upgrade_to_agent_v7
+[2]: /agent/guide/python-3
+[3]: /agent/guide/agent-v6-python-3
+{{% /tab %}}
+{{% tab "Agent v6 vs v5" %}}
+
+**Agent version 6 main changes**:
+
+The big difference between Agent 5 and Agent 6 is that Agent 6 is a complete rewrite of the core Agent in Golang. Golang has allowed the Agent to take advantage of concurrency. In place of the three processes the Agent v5 used to run—*the Forwarder*, *the Collector*, and *DogStatsD*—there is now only one process: *the Agent*. It also comes with a number of other core improvements:
+
+* Agent v6 has significantly improved resource usage over Agent v5:
+  * It has decreased CPU usage
+  * It has decrease memory usage
+  * It uses fewer file descriptors
+  * It has an all around decreased footprint
+
+* Agent 6 uses [two additional ports][1]:
+    * `5000` to expose its runtime metrics.
+    * `5001` for the [Agent CLI/GUI commands][2].
+
+    **Note**: You can specify different ports for `expvar_port` and `cmd_port` in the `datadog.yaml` file.
+
+* Custom build your Agent v6 and [DogStatsD][3] much easier and with many more configuration options, to include or exclude almost anything.
+
+**Agent v6 new functionalities**:
```

Comment:
Nit: maybe `features` instead of `functionalities`? Feel 100% free to discard.

Reply:
let's keep functionalities for now